### PR TITLE
set filter form theme

### LINKED ISSUE #3332: Batch action breaks when coming from list view with filter using `doctrine_orm_model_autocomplete`
Coming from a regular LIST view with URL
`admin/employmentcontract/list`
without further filtering I mark some rows and submit the `delete` batch action
which leads me to the following URL:
`admin/employmentcontract/batch?filter[_sort_order]=ASC&filter[_sort_by]=id&filter[_page]=1&filter[_per_page]=32`

This is the expected behaviour.

When coming from the same list but using a `doctrine_orm_model_autocomplete` filter I get this URL:
`admin/employmentcontract/batch?filter[_sort_order]=ASC&filter[_sort_by]=id&filter[_page]=1&filter[_per_page]=32&filter[branch][type]=&filter[branch][value]=2`

The batch action breaks with:
**Twig_Error_Runtime: An exception has been thrown during the rendering of a template ("Notice: Array to string conversion") in form_div_layout.html.twig at line 13.**

Any idea @pulzarraider ?
* https://github.com/sonata-project/SonataDoctrineORMAdminBundle/pull/388

### PR DESCRIPTION
**Pull request by radmar**
Closes  #3332
### Changelog

``` markdown
### Fixed
- Batch action breaks when coming from list view with filter using `doctrine_orm_model_autocomplete`
```

### CODE REVIEW

**Comment by Soullivaneuh** [comment]
Please update your PR body following this template: https://github.com/sonata-project/SonataAdminBundle/blob/3.x/.github/PULL_REQUEST_TEMPLATE.md

**Comment by radmar** [comment]
PR body updated

**Comment by Soullivaneuh** [comment]
We need tests that proof the bug relevated in #3332.

Could you please work on it?

**Comment by core23** [comment]
Maybe we could extract this to a method? So others can use this?

**Comment by Soullivaneuh** [comment]
@core23 Could you please elaborate?

**Comment by core23** [comment]
Setting a form theme to a form could be used somewhere else, so we should create a simple method which is doing this.

**Comment by OskarStark** [comment]
@radmar could you please apply the comments?

**Comment by core23** [comment]
Ping @radmar

**Comment by OskarStark** [comment]
Stuck Style CI build @Soullivaneuh

**Comment by core23** [comment]
IMO we should set the filter theme in the twig templates, so the user have a chance to customize it.

**Comment by core23** [comment]
Please rebase and use `setFormTheme` @radmar

File: Controller/CRUDController.php [hunk 1]
```diff
@@ -409,6 +409,7 @@ public function batchAction()
             $actionLabel = $this->admin->trans($batchActions[$action]['label'], array(), $translationDomain);
 
             $formView = $datagrid->getForm()->createView();
+            $this->get('twig')->getExtension('form')->renderer->setTheme($formView, $this->admin->getFilterTheme());
```
Comment (core23) [inline Controller/CRUDController.php]: Please use `setFormTheme`

**Comment by OskarStark** [comment]
@core23 i created a new PR: #4390

**Pull request closed without merging**